src/igvfd/schemas/alignment_file.json

```diff
@@ -75,6 +74,39 @@
                     ]
                 }
             }
+        },
+        "content_type": {
+            "comment": "Only methylated reads do not require reference files.",
+            "oneOf": [
+                {
+                    "required": [
+                        "reference_files"
+                    ],
+                    "properties": {
+                        "content_type": {
+                            "enum": [
+                                "alignments",
+                                "alignments with modifications",
+                                "transcriptome alignments"
+                            ]
+                        }
+                    }
+                },
+                {
+                    "not": {
```

Comment:
is this dependency saying if content_type is methylated reads, you can't specify reference_files? which is more strict than not required but it's fine?